### PR TITLE
Small improvements for mod description

### PR DESCRIPTION
**Pull request by jakobharder**

### CODE REVIEW

**Comment by taubenangriff** [comment]
I think then we should do it with prefixes entirely. 

Either you link a file, or you link markdown, both should be specified mutually exclusive. Also, I don't really like that you have to specify a heading first, just in case you want an image as the opener.

so your description becomes something like:

`md::# My Mod Header\n\n This is a mod description` in case of markdown 

or 

`file::my_mod.md` to embed a file, especially useful if we are talking huge descriptions in like all languages that just waste memory.

File: ModManager/Views/Components/ModDescriptionDisplay.xaml.cs [hunk 1]
```diff
@@ -147,16 +147,21 @@ public void SetDisplayedMod(Mod? mod)
...
-                UseMarkdownDescription = exists;
-                MarkdownDescription = exists ? File.ReadAllText(full_path!) : null;
-            }
+            string tryDescAsPath = Path.Combine(Mod.FullModPath, description);
+            if (tryDescAsPath.EndsWith(".md") && File.Exists(tryDescAsPath))
+                MarkdownDescription = File.ReadAllText(tryDescAsPath);
+
+            // assume it's markdown if it starts with a level 1 header
+            if (description.StartsWith("# "))
+                MarkdownDescription = description;
```
Comment (taubenangriff) [inline ModManager/Views/Components/ModDescriptionDisplay.xaml.cs:162]:
```suggestion
            if (description.StartsWith("file::")
            {
                 string descAsPath = Path.Combine(Mod.FullModPath, description.Substring(6));
                 if(File.Exists(descAsPath))
                     MarkdownDescription = File.ReadAllText(tryDescAsPath);
            }
            else if (description.StartsWith("md::"))
                MarkdownDescription = description.Substring(4);
```

**Review by taubenangriff — COMMENTED** [review]
prefixes could look like this

**Comment by jakobharder** [comment]
I knew you wouldn't like it 😄
Prefix it is then.

**Pull request merged**